Makefile

```diff
@@ -5,6 +5,10 @@ DATA_FILES := $(shell find conf | sed 's/ /\\ /g')
 LESS_FILES := $(wildcard public/less/gogs.less public/less/_*.less)
 GENERATED  := pkg/bindata/bindata.go public/css/gogs.css
 
+ADOCJS		 := node_modules/asciidoctor.js/dist/asciidoctor.min.js
```

Comment:
Do not check in node_modules, only import final .min.js and .min.css under `public/plugins` directory. Look for plugins already under that directory for example.